### DISCUSSION TITLE
Add Data Class/Model Generator For Kotlin Development or Flutter Development

### DISCUSSION THREAD

**Discussion by rmaprojects** [supabase/supabase#22866] [Feature Requests]
I realised that Supabase for Kotlin is kinda tricky because I have to convert the Table into Serialized data class. I know I can generate the whole data class from JSON, but it seems complicated since I have to copy-paste the API key repeatedly from Supabase into API Tester like Postman etc.

It would be cooler if Supabase has JSON to Model kinda like feature for Flutter/Kotlin Development

**Comment by mmvergara** [comment]
Hey for flutter, i made a tool'ish to [Generate Dart classes from your Supabase schema](https://github.com/mmvergara/supadart) , maybe it can help you..


### Fetching Data Example

```dart
// fetchedBooks is a typeof List<Books>
final fetchedBooks = await supabase
      .from(Books.table_name)
      .select("*")
      .withConverter((data) => data.map(Books.fromJson).toList());
```
